tutorials/poststack.py

```diff
@@ -277,3 +277,6 @@
     axs[5][2].imshow(minv3d_reg[:, :, 220].T, cmap='gist_rainbow', vmin=m.min(), vmax=m.max())
     axs[5][2].set_title('Regularized Inverted Model iter x-y')
     axs[5][2].axis('tight')
+
+###############################################################################
+# To run this tutorial with our NCCL backend, refer to :ref:`sphx_glr_tutorials_poststack_nccl.py`
```

Comment:
@tharittk I worry this may be broken in the online documentation as we don't build `tutorials_poststack_nccl`... I'll let it render and see, we can always make a small PR later to fix it :)